blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/io/IntegrationTestBase.java

```diff
@@ -270,8 +275,64 @@ protected void generateEnumRollups(Locator locator, long from, long to, Granular
         AstyanaxWriter.getInstance().insertRollups(writeContexts);
     }
 
+    private static final String TENANT1 = "11111";
+    private static final String TENANT2 = "22222";
+    private static final long   DELTA_MS = 2000;
+
+    protected Range getRangeFromMinAgoToNow(int deltaMin) {
+        // ask for 5 minutes back
+        long now = System.currentTimeMillis();
+        return new Range(now - (deltaMin*60*1000), now);
+    }
+
+    protected Map<Locator, List<IMetric>> generateEnumForTenants() throws Exception {
+        Map<Locator, List<IMetric>> locatorToMetrics = new HashMap<Locator, List<IMetric>>();
+
+        // setup some Enum data
+        long startTime = System.currentTimeMillis();
+        final Locator locator1 = Locator.createLocatorFromPathComponents(TENANT1, "my", "enum", "values");
```

Comment:
These metrics are being deleted after every test. But I can add test name for clarity.